### ISSUE TITLE
Multi-Schema Reference Field UI Broken

### ISSUE THREAD

**Issue by kitschpatrol** [local-projects/lp.squidex-cms#11]
If you assign more than one schema validation constraint to a reference field, the content-edit UI runs into trouble.

Say you have a field like this, which is supposed to permit references to content with any of three schema types:
![Screen Shot 2020-03-30 at 11 25 46 PM](https://user-images.githubusercontent.com/194164/77984098-aa148880-72de-11ea-892d-c8d47635fad2.png)
With the reference field's editing view set to "Dropdown" or "Tag", you **will not** see the reference at all if it's not of the first type in the field's schema constraint list.

----

Here's a dropdown example. In the schema edit view, the field's editor view is set to dropdown:
![Screen Shot 2020-03-30 at 11 36 12 PM](https://user-images.githubusercontent.com/194164/77984324-4048ae80-72df-11ea-9ac5-48391787fee6.png)

Then in content edit, nothing appears. Even though this element **does** internally bear a reference of the "solution" schema type:
![Screen Shot 2020-03-30 at 11 37 45 PM](https://user-images.githubusercontent.com/194164/77984412-77b75b00-72df-11ea-9c4f-bbbaf9303e08.png)

----

And the tag example. In the schema edit view, the field's editor view is set to tags:
![Screen Shot 2020-03-30 at 11 38 25 PM](https://user-images.githubusercontent.com/194164/77984445-8ef64880-72df-11ea-9506-694700b9961a.png)

In content edit, nothing:
![Screen Shot 2020-03-30 at 11 40 14 PM](https://user-images.githubusercontent.com/194164/77984536-cf55c680-72df-11ea-9c79-5de6c8a033b5.png)

----

With the reference field's editing view set to "List", you **will** see the reference even if it's not of the first type in the constraint list... but it's impossible to add or edit references unless they're of the first type. Apparently this UI is filtering content, and only allowing content of the first type in the constraint list to appear.

In the schema edit view, the field's editor view is set to list:
![Screen Shot 2020-03-30 at 11 33 15 PM](https://user-images.githubusercontent.com/194164/77984173-d6c8a000-72de-11ea-8844-a74cbdb04de8.png)

In a the content edit view, you can see the reference (because it was set via an API call, not the UI), but it's impossible to add any references through the UI that aren't of the first schema type in the validation constraint list:
![Screen Shot 2020-03-30 at 11 28 36 PM](https://user-images.githubusercontent.com/194164/77984135-c0badf80-72de-11ea-8e3f-79bd0f3df429.png)

Summary:
Seems like the UI is more or less ignoring all but the first type a reference field's schema validation list.

**Comment by SebastianStehle** [comment]
Thats true, this feature only works for List editor. I don't want to download the contents for all schemas when working with other editors.
